classes/Customer.php

```diff
@@ -186,7 +186,7 @@ class CustomerCore extends ObjectModel
             'optin' => array('type' => self::TYPE_BOOL, 'validate' => 'isBool'),
             'website' => array('type' => self::TYPE_STRING, 'validate' => 'isUrl'),
             'company' => array('type' => self::TYPE_STRING, 'validate' => 'isGenericName'),
-            'siret' => array('type' => self::TYPE_STRING, 'validate' => 'isSiret'),
+            'siret' => array('type' => self::TYPE_STRING, 'validate' => 'isGenericName'),
```

Comment:
Ok, so maybe add a maxlength 14 ? because, it's the max on db_structure..

@vincentbz why do not apply any verification ?!!